x/distribution/src/keeper/mod.rs

```diff
@@ -87,11 +93,12 @@ impl<
         ctx: &mut InitContext<'_, DB, SK>,
         genesis: GenesisState,
     ) -> anyhow::Result<()> {
-        self.set_fee_pool(ctx, &genesis.fee_pool);
+        self.set_fee_pool(ctx, &genesis.fee_pool).unwrap_gas();
```

Comment:
There is huge number of `unwrap_gas` in PR

Reply:
I've used it only from `InitContext`, `BlockContext` and `QueryContext` methods. The unwraps are legal here.